docs/api-docs/management-apis/email-templates/email-template-examples.md

```diff
@@ -11,83 +11,28 @@
-
-Go to **Transactional Emails> Email Templates**. Click **...** next to **Password Reset** and select **Edit Template**. Change the default message in the password reset template by updating the **Phrase value**. To add new text, click the **Add new** button. Enter the phrase name and phrase value. See the **Password reset** example phrases and values below.
-
-|Phrase name | Phrase value|
-|-|-|
-|message| We are sending you this email because you requested a password reset. Click on this link to create a new password.|
-|security| If you didn't request a password reset, you can ignore this email. Your password has not changed.|
-
-Next, click the **Code** tab and paste in the new {security} phrase after the shop image. For this example, you can paste 
-
-` <p>{{lang 'security'}}</p>` 
-
-into the password reset Content editor as shown below.
+To change existing text in an email template, you will need to update information in the **Phrases** and **Code** tabs. For detailed instructions, see [Editing Template Phrases](https://support.bigcommerce.com/s/article/Customizing-Emails?language=en_US#phrases). 
 
-```html
-<table class="container">
-  <tr>
-    <td>
-      <table class="row">
-        <tr>
-          <th class="column">
-            <table>
-              <tr>
-                <th>
-                  <table class="delimiter">
-                    <tr>
-                     <th>
-                       <hr>
-                </th>
-                     <th class="delimiter__image-block">
-                        <img src="{{store.cdn_path}}/img/emails/shop.png" alt="Shop image" class="delimiter__image">
-                     </th>
-                     <p>{{lang 'security'}}</p>
-                     <th>
-                      <hr>
-                     </th>
-                   </tr>
-                 </table>
-                </th>
-              </tr>
-              <th class="expander"></th>
-          </tr>
-         </table>
-        </td>
-   </tr>
- </table>
-```
-Click on the **Preview** tab to view your changes.
-[Image]()
 
 ## Adding a button
 
-To add a button to an email, you will need to add additional code to your template. Here is an example of HTML code that will add a blue button with white text to the account created template.
-
-<div class="HubBlock--callout">
-<div class="CalloutBlock--info">
-<div class="HubBlock-content">
-NOTE: You can select a different button background and text color. We suggest adhering to the minimum color contrast standards set by [WCAG](https://webaim.org/articles/contrast/). 
-</div>
-</div>
-</div> 
-
+To add a button to an email, you will need to add additional code to your template. Here is example HTML code that will add a blue button with white text to the account created template.
```

Comment:
To add a button to an email, you will need to add additional code to your template. Below is example HTML code that adds a blue button with white text to the account created template.